m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md

```diff
@@ -1,3 +1,51 @@
 ## Group Three's Session Plan
 
 M4 Student Leaders: Lauren, Amy, Dan, Jason, Chelle
+
+
+# Session 1   3/17/17
+
+### ADDITIONAL TOPICS FOR DISCUSSION THAT WERE GLOSSED OVER:
+
+The automation gear up seemed to be presented i a gloomy-doomy light--maybe could lighten the tone with more dissection
+of the topic. (See the second GENERAL SESSION OUTLINE)
+
+Just because you CAN doesn't necessarily mean you SHOULD.--needs addressing in order for us, as techies, to be empathetic and 
+aware of what's happening.
+
+What is it going to do to the quality of life of not the two ends of the financial spectrum, but the middle-class?
+
+Environmental impact of making and maintaining robots?
+
+### GENERAL SESSION OUTLINE REFORMATS:
+
+Would have like more time to mull it over solo and writing time.
+
+Sending out the reading a day in advance.  Helps facilitate discussion.
+
+Need to find an article that is a polar opposite view on the material presented in the original gear up.
```

Comment:
I found two, not sure why they aren't merged. 
http://www.economicshelp.org/blog/6717/economics/the-luddite-fallacy/ a brief introduction on the Luddite Fallacy, and the wikipedia on the Post-Scarcity economy theory https://en.wikipedia.org/wiki/Post-scarcity_economy